wisdem/drivetrainse/drive_components.py

```diff
@@ -1086,6 +1089,7 @@ def setup(self):
         self.add_output("rna_I_TT", np.zeros(6), units="kg*m**2")
 
     def compute(self, inputs, outputs, discrete_inputs, discrete_outputs):
+        logger.warning('In compute RNA_Adder')
```

Comment:
There are a few of these- are they necessary?

Reply:
These debug statements are an attempt to figure out where my Kestrel runs have been hanging. I thought it might be related to the MPI changes, but my runs appear to spend a long time solving the rotor/drivetrain coupling.  I don't have this issue locally.  

```
=====================
comp.wisdem.wt.wt_rna
=====================
In compute RNA_Adder
[00mNL: NLBGS 1 ; 3.93868928e+11 1
In compute RNA_Adder
slurmstepd: error: *** JOB 6154108 ON x3000c0s25b0n0 CANCELLED AT 2024-11-26T17:03:05 DUE TO TIME LIMIT ***
``` 

I pushed these changes because I wanted to keep them, but also re-build my WISDEM on Kestrel.  That didn't seem to help.  I am planning to revert these when ready to merge.

My next avenue is to fix the broken tests on this branch, and after that, ask you for ideas.

But since we're here, do you have any ideas/experience with an issue like this?